platforms/documentation/docs/src/docs/userguide/authoring-builds/logging.adoc

```diff
@@ -93,7 +93,7 @@ It's advisable to discuss this with your organization's security team.
 
 Some CI providers attempt to redact sensitive credentials from logs, but this process is not foolproof and typically only redacts exact matches of pre-configured secrets.
 
-If you suspect that a Gradle Plugin may inadvertently expose sensitive information, please contact security@gradle.com for assistance with disclosure.
+If you suspect that a Gradle Plugin may inadvertently expose sensitive information, please contact mailto:security@gradle.com[our security team] for assistance with disclosure.
```

Comment:
💅 I think we shouldn't close 
* https://github.com/gradle/gradle/issues/31156

just by fixing the link. The check should also be updated, as Louis suggested. (It's totally fine to do it later in a separate PR, though)